.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish cpu dockers
+on:
+  workflow_dispatch
```

Comment:
Trigger on Tags?

Reply:
Good suggestion. We could have a special docker release tag if we don't want the regular release tag to trigger this for some reason.

Reply:
As we discussed in the meeting, I'm against automatic triggering of this as it would be useless. For each publish, image files should be checked if they are working, dependencies should be checked, newer version of OS should be added or unsupported version should be removed. So, instead of automatic trigger, for each publish we should have a separate PR that checks for these and makes the necessary updates to the files and then after merging that PR with develop, this can be triggered from develop branch actions. 

We also we want to make sure that the release is completely successful (no hot bugs) before updating the docker image registry because we don't want to break any application that uses those packages.

I'm not against adding automatic Tags based on version strings tho. However, if we are going to check everything manually before publishing, I don't see a point for going that way either.

Reply:
As discussed we will trigger this manually.